ajax/libs/recorderjs/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "recorderjs",
+  "version": "0.1.0",
+  "description": "A plugin for recording/exporting the output of Web Audio API nodes",
+  "filename": "recorder.min.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/mattdiamond/Recorderjs.git"
+  },
+  "author": "Matt Diamond",
```

Comment:
Would be good if http://matt-diamond.com is added (it's open to the public but the email is only open those who login GitHub) Ref: https://github.com/mattdiamond.